.github/workflows/ci.yml

```diff
@@ -116,6 +116,8 @@ jobs:
         run: pnpm install
       - name: Build frontend
         working-directory: ./frontend
+        env:
```

Comment:
since this is building the docker frontend image I don't think we need to set VITE_MONDEY_API_URL here